packages/cli-packages/storybook-vite/src/commands/storybookHandler.ts

```diff
@@ -0,0 +1,157 @@
+import fs from 'node:fs'
+import path from 'node:path'
+
+import type { ExecaError } from 'execa'
+import execa from 'execa'
+
+import { getPaths } from '@redwoodjs/project-config'
+// Allow import of untyped package
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { errorTelemetry } from '@redwoodjs/telemetry'
+
+import c from '../lib/colors'
+import type { StorybookYargsOptions } from '../types'
+
+/*
+readFile and writeFile are somewhat duplicated from @redwoodjs/cli; I could not for the life of me get the package
```

Comment:
`@redwoodjs/cli` is a bit strange. It's not ment to be imported from. Stuff that should be shared goes in `cli-helpers`. So maybe `readFile` and `writeFile` should live there. But since you modified `writeFile` anyways they might as well both live here, at least for now 🙂

Reply:
ahhhh got it, good to know — and sounds good :)